content/nomad/v1.11.x/content/docs/job-declare/strategy/singleton.mdx

```diff
@@ -0,0 +1,300 @@
+---
+layout: docs
+page_title: Configure singleton deployments
+description: |-
+  Declare a job that guarantees only a single instance can run at a time, with
+  minimal downtime.
+---
+
+# Configure singleton deployments
+
+A singleton deployment is one where there is at most one instance of a given
+allocation running on the cluster at one time. You might need this if the
+workload needs exclusive access to a remote resource like a data store. Nomad
+does not support singleton deployments as a built-in feature. Your workloads
+continue to run even when the Nomad client agent has crashed, so ensuring
+there's at most one allocation for a given workload some cooperation from the
+job. This document describes how to implement singleton deployments.
+
+## Design Goals
+
+The configuration described here meets two primary design goals:
+
+* The design will prevent a specific process with a task from running if there
+  is another instance of that task running anywhere else on the Nomad cluster.
+* Nomad should be able to recover from failure of the task or the node on which
+  the task is running with minimal downtime, where "recovery" means that the
+  original task should be stopped and that Nomad should schedule a replacement
+  task.
+* Nomad should minimize false positive detection of failures to avoid
+  unnecessary downtime during the cutover.
+
+There's a tradeoff between between recovery speed and false positives. The
+faster you make Nomad attempt to recover from failure, the more likely that a
+transient failure causes a replacement to be scheduled and a subsequent
+downtime.
+
+Note that it's not possible to design a perfectly zero-downtime singleton
+allocation in a distributed system. This design will err on the side of
+correctness: having 0 or 1 allocations running rather than the incorrect 1 or 2
+allocations running.
```

Comment:
```suggestion
allocation in a distributed system. This design errs on the side of
correctness: having zero or one allocations running rather than the incorrect one or two
allocations running.
```
nit [spell out numbers zero thru nine](https://github.com/hashicorp/web-unified-docs/blob/main/docs/style-guide/numbers-dates-time/words-as-numbers.md)